boot/seal.go

```diff
@@ -61,10 +61,10 @@ var (
 
 // FDESetupHookParams contains the inputs for the fde-setup hook
 type FDESetupHookParams struct {
-	Key     *secboot.EncryptionKey
+	Key     secboot.EncryptionKey
 	KeyName string
 
-	Model *asserts.Model
+	Models []*asserts.Model
```

Comment:
to be clear, we will use multiple models when we go to implement remodeling correct?

and when we implement remodelling we will use a different/new function from boot/seal.go, because all of the current helpers afaict only take a single model argument

Reply:
yes, remodeling would be a main use case